common/go.mod

```diff
@@ -183,6 +182,7 @@ require (
 	github.com/rjeczalik/notify v0.9.1 // indirect
 	github.com/rs/cors v1.7.0 // indirect
 	github.com/russross/blackfriday/v2 v2.1.0 // indirect
+	github.com/scroll-tech/da-codec v0.0.0-20240730031611-1b736159d5cb // indirect
```

Comment:
update da-codec tag version

Reply:
will update this dependency with final version once da-code pr is merged.

Reply:
currently only using pseudo-version, we can release versions like `v0.0.1` after da-codec repo's refactoring pr is merged.